Userland/Utilities/js.cpp

```diff
@@ -1041,6 +1046,24 @@ void ReplObject::initialize_global_object()
     define_native_function("load", load_file, 1, attr);
     define_native_function("save", save_to_file, 1, attr);
     define_native_function("loadJSON", load_json, 1, attr);
+
+    define_native_accessor(
+        "_",
+        [](JS::VM&, JS::GlobalObject&) {
+            return last_value.value();
+        },
+        [](JS::VM& vm, JS::GlobalObject& global_object) -> JS::ThrowCompletionOr<JS::Value> {
+            VERIFY(is<ReplObject>(global_object));
+            dbgln("Disable writing last value to '_'");
```

Comment:
This should probably be outln to also work on serenity?

Reply:
I have made it output with `warnln` if we're in serenity

Reply:
Not sure I understand why we need to make it this special - everything else in the REPL also just uses `outln()`?

Reply:
Well I was thinking you don't want it to output to stdout in case your outputting to a file but I guess we're in a REPL so that doesn't matter. (And the ifdef to not output twice on lagom)

But I'll just make it use outln :+1: